x-pack/test/detection_engine_api_integration/security_and_spaces/tests/update_rules.ts

```diff
@@ -175,11 +179,213 @@ export default ({ getService }: FtrProviderContext) => {
             },
           },
         ];
+        // This should be "no_actions"
         outputRule.throttle = '1m';
         const bodyToCompare = removeServerGeneratedPropertiesIncludingRuleId(body);
         expect(bodyToCompare).to.eql(outputRule);
       });
 
+      it('should update a single rule property and remove the action', async () => {
+        const [connector1] = await Promise.all([
+          supertest
+            .post(`/api/actions/connector`)
+            .set('kbn-xsrf', 'foo')
+            .send({
+              name: 'My action',
+              connector_type_id: '.slack',
+              secrets: {
+                webhookUrl: 'http://localhost:1234',
+              },
+            }),
+        ]);
+
+        const action1 = {
+          group: 'default',
+          id: connector1.body.id,
+          action_type_id: connector1.body.connector_type_id,
+          params: {
+            message: 'message',
+          },
+        };
+
+        const ruleWithConnector: ReturnType<typeof getSimpleRule> = {
+          ...getSimpleRule('rule-1'),
+          actions: [action1],
+        };
+        const createdRule = await createRule(supertest, ruleWithConnector);
+        expect(createdRule.actions.length).to.eql(1);
```

Comment:
I ensure the actions array exists with the rule to start with before doing an update to remove the action.